Assets/Scripts/RandomPotionSpawner.cs

```diff
@@ -0,0 +1,60 @@
+using UnityEngine;
+
+public class RandomPotionSpawner : MonoBehaviour
+{
+    [SerializeField]
+    GameObject healthPotion;
```

Comment:
Pour que ce soit mieux typé on pourrait utiliser le type `Potion` ici : 
```suggestion
    Potion healthPotion;
```

Ensuite au moment d'instancier la potion on ferait juste : 
```
Instantiate(potionToSpawn.gameObject, spawnPosition, Quaternion.identity);
```